collect_app/src/main/java/org/odk/collect/android/formmanagement/BlankFormsListViewModel.java

```diff
@@ -0,0 +1,78 @@
+package org.odk.collect.android.formmanagement;
+
+import androidx.annotation.NonNull;
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+import androidx.lifecycle.ViewModelProvider;
+
+import org.odk.collect.android.forms.FormRepository;
+import org.odk.collect.android.forms.MediaFileRepository;
+import org.odk.collect.android.openrosa.api.FormAPI;
+import org.odk.collect.android.openrosa.api.FormAPIError;
+import org.odk.collect.android.utilities.MultiFormDownloader;
+import org.odk.collect.async.Scheduler;
+
+import javax.inject.Inject;
+
+public class BlankFormsListViewModel extends ViewModel {
```

Comment:
I think eventually this should also encapsulate `DiskSyncTask`, hold the list of forms that the adapter needs (probably as `LiveData`) and be responsible for sorting.